webapp/src/app/core/modules/openapi/model/pull-request-label.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Hephaestus API
+ * API documentation for the Hephaestus application server.
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * Contact: felixtj.dietrich@tum.de
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+export interface PullRequestLabel { 
+    name?: string;
```

Comment:
Is there a way to make it not optional? I think if it exists it is always defined with `name` and `color`

Reply:
This seems to be a general issue with JPA as OpenAPI Java-Variables can always be null implicitly. It applies to all our other model interfaces too.